lib/main.dart

```diff
@@ -131,6 +132,13 @@ Future<void> main(List<String> args) async {
   final gitHub = GitHub();
   di.registerSingleton<GitHub>(gitHub);
 
+  final systemTray = SystemTray();
```

Comment:
This would prbly break in android so do

```dart
if (!isMobile) {
    final systemTray = SystemTray();
    await systemTray.init();
    di.registerSingleton<SystemTray>(
      systemTray,
      dispose: (s) async => s.dispose(),
    );
  }
```